src/main/java/org/opensearch/commons/authuser/User.java

```diff
@@ -250,4 +253,12 @@ public List<String> getCustomAttNames() {
     public String getRequestedTenant() {
         return requestedTenant;
     }
+
+    public boolean isSuperUser(Settings settings) {
```

Comment:
nit: should this be named ``isAdminUser`` or ``isAdminDn``

Reply:
Changed to `isAdminDn`. admin user may be confusing since there could be different levels of admin. AdminDn implies the user connecting with admin certificate.